File: UNIDAD 5/C#/U5_P4.cs
using System;

namespace U5_P4
{
    internal class U5_P4
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese un número: ");
            int n = Convert.ToInt32(Console.ReadLine());

            if (EsPrimo(n))
            {
                Console.WriteLine(n + " es un número primo.");
            }
            else
            {
                Console.WriteLine(n + " no es un número primo.");
            }
        }

        public static bool EsPrimo(int n)
        {
            if (n <= 1)
            {
                return false;
            }

            for (int i = 2; i <= Math.Sqrt(n); i++)
            {
                if (n % i == 0)
                {
                    return false;
                }
            }

            return true;
        }
    }
}

File: UNIDAD 5/JAVA/U5_P4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u5_p4;
import java.util.Scanner;
/**
 *
 * @author fatim
 */
public class U5_P4 {

    public static void main(String[] args) {
      Scanner L = new Scanner(System.in);
   System.out.print("Ingrese un número: ");
        int n = L.nextInt();

        if (Primo(n)) {
            System.out.println(n + " es un número primo.");
        } else {
            System.out.println(n + " no es un número primo.");
        }

    }

    public static boolean Primo(int n) {
        if (n <= 1) {
            return false;
        }

        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }

        return true;
    }
}
